framework/widgets/Pjax.php

```diff
@@ -99,13 +99,13 @@ class Pjax extends Widget
     public $clientOptions;
     /**
      * @inheritdoc
-     * Note: This variable is redeclared to force separate counting for Pjax widgets (issue #12969)
-     * @since 2.0.11
+     * @internal
+     * @since 2.0.11 (#12969)
```

Comment:
No need for `@since` tag here. API isn't new and it's not actually `@internal`. It could be set by end user and that's what we're doing in tests.